src/test/kotlin/uk/gov/dluhc/printapi/service/PrintRequestsServiceTest.kt

```diff
@@ -86,15 +117,22 @@ class PrintRequestsServiceTest {
         printRequestsService.processPrintRequests(batchSize)
 
         // Then
-        verify(printDetailsRepository).getAllByStatus(Status.PENDING_ASSIGNMENT_TO_BATCH)
-        verify(idFactory).batchId()
-        verify(processPrintRequestQueue).submit(expectedMessage)
+        verify(printDetailsRepository).getAllByStatus(PENDING_ASSIGNMENT_TO_BATCH)
+        verify(certificateRepository).findByStatusIs(PENDING_ASSIGNMENT_TO_BATCH)
+        verify(idFactory, times(2)).batchId()
+        verify(processPrintRequestQueue, times(2)).submit(expectedMessage)
 
         verify(printDetailsRepository).save(capture(savedRecordArgumentCaptor))
         val savedPrintDetails = savedRecordArgumentCaptor.value!!
         assertThat(savedPrintDetails.printRequestStatuses!!.sortedBy { it.eventDateTime }.map { it.status })
-            .containsExactly(Status.PENDING_ASSIGNMENT_TO_BATCH, Status.ASSIGNED_TO_BATCH)
```

Comment:
Did you only mean to do the static import on `PENDING_ASSIGNMENT_TO_BATCH` or is it worth doing `ASSIGNED_TO_BATCH` as well?

Reply:
Imported